tests/dynamic_data/publications/crs/vectors/__init__.py

```diff
@@ -101,18 +101,21 @@ def generate(workspace=None):
 
         for rest_param_dict in rest_param_dicts:
             wms_spacial_precision_assert = [Action(publication.geoserver.wms_spatial_precision, {
-                'epsg_code': epsg_code,
+                'epsg_code': wms_epsg_code,
                 'extent': extent,
                 'img_size': img_size,
                 'wms_version': wms_version,
                 'diff_line_width': diff_line_width,
-                'obtained_file_path': f'tmp/artifacts/test_spatial_precision_wms/sample_point_cz_{style_type}_{epsg_code}{suffix}.png',
-                'expected_file_path': f'{DIRECTORY}/sample_point_cz_{epsg_code}{suffix}.png',
+                'obtained_file_path': f'tmp/artifacts/test_spatial_precision_wms/sample_point_cz_{style_type}_{wms_epsg_code}{suffix}.png',
+                'expected_file_path': f'{DIRECTORY}/sample_point_cz_{wms_epsg_code}{suffix}.png',
             })
-                for epsg_code, extent, img_size, style_type, wms_version, diff_line_width, suffix in
+                for wms_epsg_code, extent, img_size, style_type, wms_version, diff_line_width, suffix in
                 EXP_WMS_PICTURES
-                if style_type == rest_param_dict.get('style_file')
+                if style_type == REST_PARAMETRIZATION['style_file'][rest_param_dict['style_file']]
+                # If one and only one of the CRSs is 5514, use low resolution
+                and ((wms_epsg_code == 5514) != (epsg_code == 5514)) == (suffix == '_low')
```

Comment:
If either wms_epsg_code or epsg_code (not both) is 5514 and style is QML, use '_low', otherwise use ''.